server/api/api.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-plugin-autolink/server/autolink"
+)
+
+const PluginIDContextValue = "pluginid"
+
+type LinkStore interface {
```

Comment:
0/5 `AutolinkStore` for consistency? I don't like the verbosity though. Here's an idea, since there's no _other_ `Store`, maybe just `Store` for now?